src/includes/functions/math/complex/src/pow.js

```diff
@@ -17,9 +17,6 @@ pow(complex) {
 	 * Im = exp(c * logsq2 - d * arg(z_1)) * sin(d * logsq2 + c * arg(z_1))
 	 *
 	 */
-	if (!complex.isComplex) {
-		throw "[TheoremJS]: Complex operation require complex numbers"
-	}
```

Comment:
Remove previous `throw err`, as we are now going to support more than just complex arguments